projects/documentation/scripts/copy-component-docs.js

```diff
@@ -57,6 +57,28 @@ const findDeclaration = (customElements, test) => {
     return declaration;
 };
 
+const findDeprecationNotice = async function (packageName) {
+    let packageJSON = {};
+    try {
+        packageJSON = await import(
+            `../../../packages/${packageName}/package.json`,
+            {
+                assert: { type: 'json' },
+            }
+        ).then((packageDefault) => packageDefault.default);
+    } catch (error) {
+        try {
+            packageJSON = await import(
+                `../../tools/${packageName}/package.json`,
+                {
+                    assert: { type: 'json' },
+                }
+            ).then((packageDefault) => packageDefault.default);
+        } catch (error) {}
+    }
+    return packageJSON.deprecationNotice;
+};
+
```

Comment:
I think we know whether something is `packages` or `tools` in a way that doesn't require this `try/catch` gate.